ClearKeep/Scenes/GroupDetail/Views/DetailContentView.swift

```diff
@@ -0,0 +1,283 @@
+					HStack(spacing: Constants.spaceHstack) {
+						Spacer()
+						ForEach(0..<member.prefix(4).count, id: \.self) { user in
+							AvatarDefault(.constant(member[user].displayName), imageUrl: member[user].avatar)
+								.frame(width: Constants.imageSize.width, height: Constants.imageSize.height)
+						}
+						Spacer()
+					}
+				} else {
+					HStack(spacing: Constants.spaceHstack) {
+						Spacer()
+						ForEach(0..<member.prefix(4).count, id: \.self) { user in
+							AvatarDefault(.constant(member[user].displayName), imageUrl: member[user].avatar)
+								.frame(width: Constants.imageSize.width, height: Constants.imageSize.height)
+						}
+						ZStack {
+							Circle()
+								.fill(foregroundButtonImage)
+								.frame(width: Constants.imageSize.width, height: Constants.imageSize.height)
+							Text("+\(member.count - 4)")
+								.font(AppTheme.shared.fontSet.font(style: .body3))
+								.foregroundColor(AppTheme.shared.colorSet.offWhite)
+						}
+						Spacer()
+					}
+				}
+				HStack(alignment: .center) {
+					ImageButtonCircleCall("General.Audio".localized, image: AppTheme.shared.imageSet.phoneCallIcon, action: audioAction).disabled(self.disableCall)
+					Spacer()
+					ImageButtonCircleCall("General.Video".localized, image: AppTheme.shared.imageSet.videoIcon, action: videoAction).disabled(self.disableCall)
+				}
+				.padding(.horizontal, Constants.paddingHorizontal)
+				ForEach(DetailType.allCases, id: \.self) { detail in
+					didSelect(detail)
+				}
+				Spacer()
+			}
+			.padding(.horizontal, Constants.padding)
+			.hiddenNavigationBarStyle()
+			.applyNavigationBarPlainStyle(title: groupData?.groupName
+										  ?? "Name",
+										  titleColor: titleColor,
+										  backgroundColors: backgroundButtonBack,
+										  leftBarItems: {
+				BackButtonStandard(customBack)
+			},
+										  rightBarItems: {
+				Spacer()
+			})
+			.background(backgroundColorView)
+			.edgesIgnoringSafeArea(.all)
+			.alert(isPresented: self.$isShowAlert) {
+				switch self.errorType {
+				case .error:
+					return Alert(title: Text("GroupChat.Warning".localized),
+								 message: Text(self.messageAlert),
+								 dismissButton: .default(Text("GroupChat.Ok".localized)))
+				case .permission:
+					return Alert(title: Text("Call.PermistionCall".localized),
+								 message: Text("Call.GoToSetting".localized),
+								 primaryButton: .default(Text("Call.Settings".localized), action: {
+						guard let url = URL(string: UIApplication.openSettingsURLString) else {
+							return
+						}
+						UIApplication.shared.open(url)
+					}),
+								 secondaryButton: .default(Text("Call.Cancel".localized)))
+				case .existCall:
+					return Alert(title: Text("General.Warning".localized),
+								 message: Text("Call.HaveExistCall".localized),
+								 dismissButton: .default(Text("General.OK".localized)))
+				}
+
+			}
+			.alert("GroupChat.Warning".localized, isPresented: $isLogout) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/GroupDetail/Views/DetailContentView.swift#L126"></span>Arguments should be either on the same line, or one per line.<br />
<code>multiline_arguments</code> <code>DetailContentView.swift:126</code></td>
    </tr>
  </tbody>
</table>